library/core/src/fmt/mod.rs

```diff
@@ -633,6 +633,23 @@ pub use macros::Debug;
 /// [tostring]: ../../std/string/trait.ToString.html
 /// [tostring_function]: ../../std/string/trait.ToString.html#tymethod.to_string
 ///
+/// # Internationalization
+///
+/// Because a type only has one `Display` implementation, it is often preferable
```

Comment:
```suggestion
/// Because a type can only have one `Display` implementation, it is often preferable
```

As phrased, it's a little unclear whether a type must have a `Display` implementation (implying that there's a default if you don't supply one), can only have one, or normally only has one (but can have multiple).

Reply:
That's a good suggestion.